barretenberg/cpp/src/barretenberg/benchmark/ultra_bench/ultra_plonk_rounds.bench.cpp

```diff
@@ -71,9 +71,9 @@ BBERG_PROFILE static void test_round(State& state, size_t index) noexcept
 // Fast rounds take a long time to benchmark because of how we compute statistical significance.
 // Limit to one iteration so we don't spend a lot of time redoing full proofs just to measure this part.
 ROUND_BENCHMARK(PREAMBLE)->Iterations(1);
-ROUND_BENCHMARK(FIRST_WIRE_COMMITMENTS);
-ROUND_BENCHMARK(SECOND_FIAT_SHAMIR_ETA);
-ROUND_BENCHMARK(THIRD_FIAT_SHAMIR_BETA_GAMMA);
-ROUND_BENCHMARK(FOURTH_FIAT_SHAMIR_ALPHA_AND_COMMIT);
-ROUND_BENCHMARK(FIFTH_COMPUTE_QUOTIENT_EVALUTION);
-ROUND_BENCHMARK(SIXTH_BATCH_OPEN);
+ROUND_BENCHMARK(FIRST_WIRE_COMMITMENTS)->Iterations(1);
+ROUND_BENCHMARK(SECOND_FIAT_SHAMIR_ETA)->Iterations(1);
+ROUND_BENCHMARK(THIRD_FIAT_SHAMIR_BETA_GAMMA)->Iterations(1);
```

Comment:
What's the rationale with making them all iterations 1? just that it takes long? (the way I did rounds was pretty inefficient but simpler with how googlebench is setup)

Reply:
I think I got impatient while working on it, will revert.